integration/smartbft/smartbft_test.go

```diff
@@ -1655,3 +1774,42 @@ func updateBatchSize(
 	}
 	nwo.UpdateOrdererConfig(network, orderer, channel, config, updatedConfig, peer, orderer)
 }
+
+// marshalOrPanic serializes a protobuf message and panics if this
+// operation fails
+func marshalOrPanic(pb proto.Message) []byte {
+	data, err := proto.Marshal(pb)
+	if err != nil {
+		panic(err)
+	}
+	return data
+}
+
+func createEnvelope(
```

Comment:
this is nice but can we please try to make a real transaction? Look at how this is done [here](https://github.com/hyperledger/fabric/blob/main/protoutil/txutils.go#L135).